TransferMethod/Sources/SelectTransferMethodTypeController.swift

```diff
@@ -88,14 +83,15 @@ final class SelectTransferMethodTypeController: UITableViewController {
 }
 
 extension SelectTransferMethodTypeController {
+    /// - Returns  transfer method count
```

Comment:
remove - from comments. In all other places as well

Reply:
Removed extra -